conference/cfp.py

```diff
@@ -300,15 +301,20 @@ class AddSpeakerToTalkForm(forms.ModelForm):
         help_text=(
             "We require a mobile phone number for all speakers "
             "for last minute contacts and in case we need "
-            "timely clarification (if no reponse to previous emails).<br>"
-            "Use the international format, eg: +39-055-123456.<br />"
-            "This number will <strong>never</strong> be published."
+            "timely clarification (if no reponse to previous emails). "
+            "Use the international format (e.g.: +44 123456789). "
+            "This field will <strong>never</strong> be published."
         ),
         max_length=30,
     )
-    gender = forms.CharField(
-        label="Type your gender",
-        max_length=32,
+    gender = forms.ChoiceField(
```

Comment:
Do we really need this field now that we have a user field for gender ? People submitting talks will have to have a user account.

Reply:
Right. I will remove it entirely.